components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -195,6 +196,10 @@ function NavigationListItem(
 
     // row action
     const action = (): void => {
+        // Don't scroll to top of page when clicking expandable items.
+        if (!disableScrollTop && !items) {
+            window.scrollTo(0, 0);
+        }
```

Comment:
Expandable drawer items can still have a navigation action (there are separate events for clicking the item and clicking the expand icon)

Reply:
I removed this comment and moved the window.scrollTop logic to be grouped with the onClick action.